wrapped-bridged-usdc/src/lib.rs

```diff
@@ -0,0 +1,145 @@
+#![no_std]
+
+elrond_wasm::imports!();
+
+const LEFTOVER_GAS: u64 = 10_000u64;
+
+#[elrond_wasm::contract]
+pub trait WrappedBridgedUsdc {
+    #[init]
+    fn init(&self, wrapped_usdc_token_id: TokenIdentifier) {
+        self.wrapped_usdc_token_id().set(wrapped_usdc_token_id);
+    }
+
+    #[only_owner]
+    #[endpoint(whitelistUSDC)]
+    fn whitelist_usdc(&self, usdc_token_id: TokenIdentifier) {
+        self.usdc_token_ids().insert(usdc_token_id);
+    }
+
+    #[only_owner]
+    #[endpoint(blacklistUSDC)]
+    fn blacklist_usdc(&self, usdc_token_id: TokenIdentifier) {
+        self.usdc_token_ids().swap_remove(&usdc_token_id);
+    }
+
+    // endpoints
+
+    #[payable("*")]
+    #[endpoint(wrapUSDC)]
+    fn wrap_usdc(
+        &self,
+        #[payment_token] payment_token: TokenIdentifier,
+        #[payment_amount] payment_amount: BigUint,
+        #[var_args] accept_funds_endpoint_name: OptionalValue<ManagedBuffer>,
```

Comment:
I think we should remove the whole accept_funds_endpoint concept, and assume the caller is always payable. Simplifies the code by a lot.

We should assume the caller is payable/payable-by-sc.